README.md

```diff
@@ -1,10 +1,97 @@
-# hello-world
-#############################################
-# this is a sample project for the purpose of understanding:
-   #  linters
-   #  gitflow
-   #  gitignore
-   #  version conrol
-   #  pull requests
-   #  make a github professional
- ##########################################
+# HTML & CSS3 Course
+
+If you are not familiar with linters, read [root level README](../README.md).
+
+## Set-up GitHub Actions
+
+Please do the following **steps in this order**:
+
+1. In the first commit of your feature branch create a `.github/workflows` folder and add a copy of [`.github/workflows/linters.yml`](.github/workflows/linters.yml) to that folder.
+   - **Remember** to use the file linked above
+   - **Remember** that `.github` folder starts with a dot.
+2. **Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students.**
+   - If you think that change is necessary - open a [Pull Request in this repository](../README.md#contributing) and let your code reviewer know about it.
+3. When you open your first pull request you should see the result of the GitHub actions checks:
+
+![gh actions checks](../assets/images/gh-actions-html-css-checks.png)
+
+Click on the `Details` link of each action to see the full output and the errors that need to be fixed:
+
+![gh actions failing checks](../assets/images/gh-actions-html-css-failing-checks.png)
+
+## Set-up linters in your local env
+
+**Note**: The `npm` package manager is going to create a `node_modules` directory to install all of your dependencies. You shouldn't commit that directory. To avoid that, you can create a [`.gitignore`](https://git-scm.com/docs/gitignore) file and add `node_modules` to it:
+
+```
+# .gitignore
+node_modules/
+```
+
+### [Lighthouse](https://developers.google.com/web/tools/lighthouse)
+
+An open-source, automated tool for improving the quality of web pages. It has audits for performance, accessibility, progressive web apps, SEO and more.
+
+You can get the Lighthouse report by any of the following ways:
+
+- [In Chrome DevTools](https://developers.google.com/web/tools/lighthouse#devtools)
+- [From the command line](https://developers.google.com/web/tools/lighthouse#cli)
+- [As a Node module](https://developers.google.com/web/tools/lighthouse#programmatic)
+- [From a web UI](https://developers.google.com/web/tools/lighthouse#psi)
+
+To access the report generated by Lighthouse on your pull request, click the `Details` link for the `Linters/Lighthouse` check and you will see the full output of the action:
```

Comment:
- [ ] Kindly use the provided MIcroverse template, as this is part of the requirements. [Template link](https://github.com/microverseinc/readme-template/blob/master/README.md)